modules/network-observability-pktdrop-overview.adoc

```diff
@@ -21,5 +21,14 @@ When packet drop tracking is enabled, you can see the following metrics represen
 * Top X dropped cause
 * Top X flow dropped rates stacked with total
 
+Two kinds of packet drops are detected by Network Observability: host drops and OVS drops. If any flows are dropped, they are shown in the side panel of the Traffic flow table. Host drops are prefixed with `SKB_DROP` and OVS drops are prefixed with `OVS_DROP`, and a link to a description of the drop type is also available in the side panel. Examples of host drop reasons are as follows:
+
+* `SKB_DROP_REASON_NO_SOCKET`: the packet dropped because the socket was not found. 
+* `SKB_DROP_REASON_TCP_CSUM`: there is a TCP checksum error.
+
+Examples of OVS drops reasons are as follows:
+
+* `OVS_DROP_LAST_ACTION`: OVS dropped because a flow was configured with an implicit drop action, for example configuring network-policy
```

Comment:
How does `OVS packets dropped due to implicit drop action, for example due to a network policy configured` sound?